lib/parity/environment.rb

```diff
@@ -90,9 +93,19 @@ def parallelize?
       arguments.include?("--parallelize")
     end
 
+    def backup_id?
+      backup_id_index = arguments.index { |arg| arg == "--backup-id" }
+      if backup_id_index && backup_id_index + 1 < arguments.length
+        arguments[backup_id_index + 1]
+      end
+    end
```

Comment:
The predicate method should return a boolean result. Given that you're using the ID returned here I think the `?` should be dropped from the method name.